active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,380 @@
+
+**Important APIs:** [PagerControl class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.pagercontrol)
+
+This control can be configured to use one of three different display modes shown below:
+
+### ComboBox Mode
+![](images/pager_control_combo.jpg)
+
+### NumberBox Mode
+![](images/pager_control_number.jpg)
+
+### NumericalButtonPanel Mode 
+![](images/pager_control_button.jpg)
+
+## Is this the right control? 
+
+Use a **PagerControl** to create a navigation interface for an end user to page through content instead of scrolling or panning. 
+
+Scrolling through all content in a single view can make it harder for the end user to find specific information and it can lead to performance issues when the content needs to be refreshed. **PagerControl** can be used as an alternate solution so the user can consume a subset of the content without needing to scroll. They can then page through the rest of the content to find what they are looking for. 
+
+## Examples
+
+A pager control has three different display modes and each display mode has configurable components within them. 
+
+### ComboBox Display Mode
+
+![](images/pager-control-combobox.png)
+
+### Pager Control Editable NumberBox Display Mode 
+
+![](images/pager-control-editable-textbox.png)
+
+### Pager Control Numerical Button Panel Display Mode
+
+![](images/pager-control-numerical-button-panel.png)
+
+A pager control does not require you to provide a max number of pages to the control. If the max number of pages is unknown, here is what each display mode will look like. 
+
+### ComboBox Display Mode
+
+(add comps)
+
+### Pager Control Editable NumberBox Display Mode 
+
+(add comps)
+
+### Pager Control Numerical Button Panel Display Mode
+
+(add comps)
+
+You can choose to customize the look of a pager control by changing the visibility of the first, previous, next, and last buttons. They can be configured to always be visible, hidden, or hidden when the selected index is the first or last page in the range. When the last page is selected, the next and last buttons will be disabled and same when the first page is selected for the first and previous page. The prefix and suffix text can also be customized to display a string you provide. 
+
+## Creating a pager control 
+
+Here is the XAML for how to add a pager control to your application using the auto display mode. Auto will choose to use the combo box mode if the NumberOfPages property is less than 11 and will show the number box mode if it is greater than 11. In this example, combo box will be chosen. 
+
+XAML
+```XAML
+<Grid>
+        <controls:DataGrid x:Name="dataGrid" 
+                Height="600" Margin="12"
+                AutoGenerateColumns="True"
+                ItemsSource="{x:Bind MyViewModel.Items}" />  
+        <muxc:PagerControl x:Name="mainPagerControl"
+                NumberOfPages="10"
```

Comment:
This doesn't look like a real scenario; how do I know to set the page count to 10? If there are 10,000 items in the DataGrid, this will set up each page to have 1000 items? Shouldn't the page count be a function of the items? Also, is this really a _max_ page count, or will it split 5 rows into 10 pages?